src/client/pydaos/raw/conversion.py

```diff
@@ -35,7 +35,15 @@ def str_to_c_uuid(uuidstr):
 
 
 def c_err_to_str(err_num):
-    """ Utility function to convert C error code to its string id """
+    """ Utility function to convert C error code to its string id.
+
+    Args:
+        err_num (int): C error code.
+
+    Returns:
+        err_str (str): String identifying a C error code.
```

Comment:
```suggestion
        str: String identifying a C error code.
```
We only print the type for the return value.

Reply:
- [ ] Remove type for returns section in docstring

Reply:
Fixed with commit 43963420b